packages/gravity/blade-runner/src/plan/env/replicant-registry.ts

```diff
@@ -0,0 +1,19 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import { type ReplicantBrain } from './interface';
+
+export class ReplicantRegistry {
+  public static readonly instance = new ReplicantRegistry();
+
+  private readonly _replicantBrains = new Map<string, ReplicantBrain<any>>();
```

Comment:
Brain is confusing. as above, don't overextend the metaphor.
Is this a stub/proxy?

Reply:
this is a class that defines the behaviour of the replicant

Reply:
@mykola-vrmchk pls add jsdoc to all things with ambigious names/meaning